### PR TITLE
open-cv-practice-intel

### PR DESCRIPTION
**Pull request by MydyakovDmitry**

### CODE REVIEW

File: modules/1_opencv/include/opencv_coins.hpp [hunk 1]
```diff
@@ -1,6 +1,5 @@
 #pragma once
 
 #include <opencv2/opencv.hpp>
-
```
Comment (allnes) [inline modules/1_opencv/include/opencv_coins.hpp:4]: Don't remove extra strings

File: modules/1_opencv/src/opencv_coins.cpp [hunk 2]
```diff
@@ -2,13 +2,37 @@
 
 using namespace cv;
 
+const double PI = 3.141592653589793238462643;
```
Comment (allnes) [inline modules/1_opencv/src/opencv_coins.cpp]: Please use CV_PI or M_PI

File: modules/1_opencv/src/opencv_coins.cpp [hunk 3]
```diff
@@ -2,13 +2,37 @@
...
+        if (contourArea(contour) / PI > 200) {
+            count_coins += 2;
+        }
+        else {
+            count_coins += 1;
+        }
+    }
+
+    return count_coins;
     CV_Error(Error::StsNotImplemented, "countCoins");
```
Comment (allnes) [inline modules/1_opencv/src/opencv_coins.cpp:37]: Please remove cv_error

File: modules/1_opencv/include/opencv_coins.hpp (hunk 1 above)
Comment (MydyakovDmitry) [reply modules/1_opencv/include/opencv_coins.hpp:4]: thank

File: modules/1_opencv/src/opencv_coins.cpp (hunk 2 above)
Comment (MydyakovDmitry) [reply modules/1_opencv/src/opencv_coins.cpp]: I made a new commit

**Pull request closed without merging**